NGitLab/Models/ReleaseCreate.cs

```diff
@@ -1,10 +1,28 @@
-﻿using System.Runtime.Serialization;
+﻿using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
 
 namespace NGitLab.Models
 {
     [DataContract]
     public class ReleaseCreate
```

Comment:
As this class is shared with the old route, won't it break it? I feel like it would be safer to create new ones for the new routes.

Reply:
Not relevant anymore if we remove the old create / update methods.

Reply:
It can share it as it doesn't modify fields, if we add new ones it'll add extra migration later on